<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Property</class>
 <widget class="QWidget" name="Property">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件属性</string>
  </property>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>文件类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="size_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>文件大小</string>
   </property>
  </widget>
  <widget class="QLabel" name="location_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>位置</string>
   </property>
  </widget>
  <widget class="QLabel" name="property_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>属性</string>
   </property>
  </widget>
  <widget class="QLabel" name="creat_time_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>创建时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="update_time_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>修改时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_content_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="size_content_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="location_content_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="property_content_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="creat_time_content_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="update_time_content_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
